services/ec2/src/main/java/org/finra/gatekeeper/services/accessrequest/delegates/RevokeAccessServiceTask.java

```diff
@@ -133,18 +93,76 @@ public void execute(DelegateExecution execution) throws Exception{
                 logger.error("Error pushing to SNS topic upon request expiration. Request ID: " + accessRequest.getId());
```

Comment:
I Think the SNS bit be helper functioned like my comment in the Grant Request class since it's just code duplication